ush/python/pygfs/task/atm_analysis.py

```diff
@@ -156,27 +153,34 @@ def finalize(self: Analysis) -> None:
         # get list of diag files to put in tarball
         diags = glob.glob(os.path.join(self.task_config.DATA, 'diags', 'diag*nc4'))
 
+        logger.info(f"Compressing {len(diags)} diag files to {atmstat}.gz")
+
         # gzip the files first
+        logger.debug(f"Gzipping {len(diags)} diag files")
         for diagfile in diags:
             with open(diagfile, 'rb') as f_in, gzip.open(f"{diagfile}.gz", 'wb') as f_out:
                 f_out.writelines(f_in)
 
         # open tar file for writing
+        logger.debug(f"Creating tar file {atmstat} with {len(diags)} gzipped diag files"
```

Comment:
Closing `)` missing.   Found this while testing `atm_analysis.py` in prufsatm.   Script failed on line 166 with `invalid syntax` error.   Added `)` and `atm_analysis.py` worked in init and run jobs.